.github/workflows/lint.yml

```
@@ -32,3 +32,20 @@ jobs:
       run: |
         git diff-index --quiet HEAD -- pwndbg tests
         ./lint.sh
+        
+    - name: Run mypy
+      uses: tsuyoshicho/action-mypy@v4
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        # Change reviewdog reporter if you need [github-pr-check,github-check,github-pr-review].
+        reporter: github-pr-check
+        # Change reporter level if you need.
+        # GitHub Status Check won't become failure with warning.
+        level: error
+        # Change the current directory to run mypy command.
+        # mypy command reads setup.cfg or other settings file in this path.
+        execute_command: source ./.venv/bin/activate && mypy pwndbg
```

1. You don't need to specify "pwndbg" at the end, it will be automatically added by the `target` option
2. The issue seems to be related to having the `source ./.venv/bin/activate &&`. I ran it locally, and this causes no output to be generated (it finishes so fast it seems like `mypy` doesn't even run), and I'm not sure why. Running it as two separate steps locally fixes the issue, but not sure how to do that here.

---

Actually, now that the dev dependencies are installed with poetry, we should be able to just do this:

```suggestion
        execute_command: poetry run mypy
```